runtime/src/main/java/com/flipkart/flux/controller/WorkFlowExecutionController.java

```diff
@@ -211,7 +212,9 @@ private void executeStates(StateMachine stateMachine, Set<State> executableState
      * @param executableStates states whose all dependencies are met
      */
     private void executeStates(StateMachine stateMachine, Set<State> executableStates, Event currentEvent) {
+        MDC.put("stateMachineId", stateMachine.getId().toString());
```

Comment:
keep stateMachineId and taskId as string constants in commons module Constants file